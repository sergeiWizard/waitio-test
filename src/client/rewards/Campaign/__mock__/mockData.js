// eslint-disable-next-line import/prefer-default-export
export const initialState = {
  filterKey: 'all',
  history: {
    action: 'PUSH',
    length: 46,
  },
  proposition: {
    count: 1,
    distance: null,
    last_created: '2020-02-04T12:32:15.446Z',
    max_reward: 0.001,
    min_reward: 0.001,
  },
};
