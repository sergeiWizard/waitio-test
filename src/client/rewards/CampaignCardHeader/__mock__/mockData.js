// eslint-disable-next-line import/prefer-default-export
export const initialState = {
  campaignData: {
    budget: 1,
    guide: {},
    is_sponsor_blacklist: false,
    max_assign_count: 1,
    name: 'Testing123',
    requiredObject: 'Testing123',
    reward: 0.001,
  },
  isDetails: false,
};

export const stateForModalDetail = {
  campaignData: {
    budget: 1,
    guide: {},
    is_sponsor_blacklist: false,
    max_assign_count: 1,
    name: 'Testing123',
    requiredObject: 'Testing123',
    reward: 0.001,
  },
  isDetails: true,
};

export const stateForTotal = {
  campaignData: {
    budget: 1,
    guide: {},
    is_sponsor_blacklist: false,
    max_assign_count: 1,
    name: 'Testing123',
    requiredObject: 'Testing123',
    reward: 0.001,
  },
  isDetails: true,
};

export const stateForModalSteem = {
  campaignData: {
    budget: 1,
    guide: {},
    is_sponsor_blacklist: false,
    max_assign_count: 1,
    name: 'Testing123',
    requiredObject: 'Testing123',
    reward: 0,
  },
  isDetails: true,
};

export const stateForModalSteemMinus = {
  campaignData: {
    budget: 1,
    guide: {},
    is_sponsor_blacklist: false,
    max_assign_count: 1,
    name: 'Testing123',
    requiredObject: 'Testing123',
    reward: -1,
  },
  isDetails: true,
};
