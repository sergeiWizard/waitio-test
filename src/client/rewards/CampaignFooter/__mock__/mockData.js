// eslint-disable-next-line import/prefer-default-export
export const initialState = {
  ownPost: false,
  pendingLike: false,
  pendingFlag: false,
  pendingFollow: false,
  pendingFollowObject: false,
  pendingBookmark: false,
  saving: false,
  onLikeClick: () => {},
  onCommentClick: () => {},
  handlePostPopoverMenuClick: () => {},
  toggleModalDetails: () => {},
};
