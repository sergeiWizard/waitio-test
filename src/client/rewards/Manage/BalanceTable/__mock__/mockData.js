export const initialState = {
  budgetTotal: {
    remaining: 6.95,
    sum_budget: 6.95,
    sum_payable: 0,
    sum_reserved: 0,
  },
  user: {
    active: {
      account_auths: [],
      key_auths: ['STM8MKLK4gtD8d2vJbzUZwvtoasxCziMiiPQFjQH8cXfr6YdKkhSF', '1'],
      weight_threshold: 1,
    },
    balance: '128.710 STEEM',
    can_vote: true,
    comment_count: 0,
    created: '2019-04-23T07:49:21',
    curation_rewards: 26127,
    delegated_vesting_shares: '0.000000 VESTS',
    downvote_manabar: {
      current_mana: '502592455076',
      last_update_time: '1583490708',
    },
    guest_bloggers: [],
    id: 1262161,
    last_account_recovery: '1970-01-01T00:00:00',
    last_account_update: '"2020-02-10T15:14:30"',
    last_owner_update: '1970-01-01T00:00:00',
    last_post: '2020-03-06T11:07:09',
    last_root_post: '2020-02-25T13:13:57',
    last_vote_time: '2020-03-06T10:31:48',
    lifetime_vote_count: 0,
    market_history: [],
    memo_key: 'STM8a1puEKRPxJRtpUeZcTEK1NyofCYNVaiNXCKPBTd3pBZn3jVcu',
    mined: false,
    name: 'asd09',
    next_vesting_withdrawal: '1969-12-31T23:59:59',
    other_history: [],
    owner: {
      account_auths: [],
      key_auths: ['STM84W9pJmUjdR3D2QaEhu1RwBvxxGcd4VeWBSq1hAVRs7ym3sZGe', 1],
      weight_threshold: 1,
    },
    pending_claimed_accounts: 0,
    post_bandwidth: 0,
    post_count: 40397,
    post_history: [],
    posting: {
      account_auths: [
        ['busy.app', 1],
        ['investarena', 1],
        ['jucci', 1],
      ],
      key_auths: ['STM6nEnuY3jRSCGB5F4cvBefRjDxDN8ProZ3166KKQCUUgFmcHD3p', 1],
      weight_threshold: 1,
    },
    posting_rewards: 678,
    proxied_vsf_votes: [0, 0, 0, 0],
    proxy: '',
    received_vesting_shares: '1997152.344711 VESTS',
    recovery_account: 'blocktrades',
    reputation: -1792344993797,
    reset_account: 'null',
    reward_sbd_balance: '0.000 SBD',
    reward_steem_balance: '0.000 STEEM',
    reward_vesting_balance: '0.000000 VESTS',
    reward_vesting_steem: '0.000 STEEM',
    savings_balance: '0.000 STEEM',
    savings_sbd_balance: '0.000 SBD',
    savings_sbd_last_interest_payment: '1970-01-01T00:00:00',
    savings_sbd_seconds: '0',
    savings_sbd_seconds_last_update: '1970-01-01T00:00:00',
    savings_withdraw_requests: 0,
    sbd_balance: '1.000 SBD',
    sbd_last_interest_payment: '2020-02-26T00:00:48',
    sbd_seconds: '777591000',
    sbd_seconds_last_update: '2020-03-06T00:00:39',
    tags_usage: [],
    to_withdraw: '265008298803',
    transfer_history: [],
    vesting_balance: '0.000 STEEM',
    vesting_shares: '13217.475594 VESTS',
    vesting_withdraw_rate: '0.000000 VESTS',
    vote_history: [],
    voting_manabar: {
      current_mana: '1765315326783',
      last_update_time: 1583494296,
    },
    voting_power: 8781,
    withdraw_routes: 0,
    withdrawn: '265008298803',
    witness_votes: [],
    witnesses_voted_for: 0,
  },
};

export const stateToOperations = {
  budgetTotal: {
    remaining: 6.95,
    sum_budget: 6.95,
    sum_payable: 50,
    sum_reserved: 50,
  },
  user: {
    active: {
      account_auths: [],
      key_auths: ['STM8MKLK4gtD8d2vJbzUZwvtoasxCziMiiPQFjQH8cXfr6YdKkhSF', '1'],
      weight_threshold: 1,
    },
    balance: '128.710 STEEM',
    can_vote: true,
    comment_count: 0,
    created: '2019-04-23T07:49:21',
    curation_rewards: 26127,
    delegated_vesting_shares: '0.000000 VESTS',
    downvote_manabar: {
      current_mana: '502592455076',
      last_update_time: '1583490708',
    },
    guest_bloggers: [],
    id: 1262161,
    last_account_recovery: '1970-01-01T00:00:00',
    last_account_update: '"2020-02-10T15:14:30"',
    last_owner_update: '1970-01-01T00:00:00',
    last_post: '2020-03-06T11:07:09',
    last_root_post: '2020-02-25T13:13:57',
    last_vote_time: '2020-03-06T10:31:48',
    lifetime_vote_count: 0,
    market_history: [],
    memo_key: 'STM8a1puEKRPxJRtpUeZcTEK1NyofCYNVaiNXCKPBTd3pBZn3jVcu',
    mined: false,
    name: 'asd09',
    next_vesting_withdrawal: '1969-12-31T23:59:59',
    other_history: [],
    owner: {
      account_auths: [],
      key_auths: ['STM84W9pJmUjdR3D2QaEhu1RwBvxxGcd4VeWBSq1hAVRs7ym3sZGe', 1],
      weight_threshold: 1,
    },
    pending_claimed_accounts: 0,
    post_bandwidth: 0,
    post_count: 40397,
    post_history: [],
    posting: {
      account_auths: [
        ['busy.app', 1],
        ['investarena', 1],
        ['jucci', 1],
      ],
      key_auths: ['STM6nEnuY3jRSCGB5F4cvBefRjDxDN8ProZ3166KKQCUUgFmcHD3p', 1],
      weight_threshold: 1,
    },
    posting_rewards: 678,
    proxied_vsf_votes: [0, 0, 0, 0],
    proxy: '',
    received_vesting_shares: '1997152.344711 VESTS',
    recovery_account: 'blocktrades',
    reputation: -1792344993797,
    reset_account: 'null',
    reward_sbd_balance: '0.000 SBD',
    reward_steem_balance: '0.000 STEEM',
    reward_vesting_balance: '0.000000 VESTS',
    reward_vesting_steem: '0.000 STEEM',
    savings_balance: '0.000 STEEM',
    savings_sbd_balance: '0.000 SBD',
    savings_sbd_last_interest_payment: '1970-01-01T00:00:00',
    savings_sbd_seconds: '0',
    savings_sbd_seconds_last_update: '1970-01-01T00:00:00',
    savings_withdraw_requests: 0,
    sbd_balance: '1.000 SBD',
    sbd_last_interest_payment: '2020-02-26T00:00:48',
    sbd_seconds: '777591000',
    sbd_seconds_last_update: '2020-03-06T00:00:39',
    tags_usage: [],
    to_withdraw: '265008298803',
    transfer_history: [],
    vesting_balance: '0.000 STEEM',
    vesting_shares: '13217.475594 VESTS',
    vesting_withdraw_rate: '0.000000 VESTS',
    vote_history: [],
    voting_manabar: {
      current_mana: '1765315326783',
      last_update_time: 1583494296,
    },
    voting_power: 8781,
    withdraw_routes: 0,
    withdrawn: '265008298803',
    witness_votes: [],
    witnesses_voted_for: 0,
  },
};
