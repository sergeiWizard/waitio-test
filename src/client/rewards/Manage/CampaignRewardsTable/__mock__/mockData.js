// eslint-disable-next-line import/prefer-default-export
export const initialState = {
  campaigns: [
    {
      _id: '5e620da2fb97af17249e3f95',
      activation_permlink: 'activate-monterey-ln7osqo19e',
      budget: 0.1,
      completed: 0,
      name: 'TstCampa',
      remaining: 1,
      reserved: 0,
      reward: 0.1,
      status: 'active',
      type: 'reviews',
    },
    {
      _id: '5e38165d31b1bb6e49b265b6',
      activation_permlink: 'activate-monterey-4znrq130xhe',
      budget: 0.2,
      completed: 0,
      name: 'capa',
      remaining: 0,
      reserved: 0,
      reward: 0.1,
      status: 'expired',
      type: 'reviews',
    },
    {
      _id: '5e3815b431b1bb6e49b265b4',
      activation_permlink: 'activate-monterey-4p0t3jnbpmr',
      budget: 0.1,
      completed: 0,
      name: 'capaz1',
      remaining: 0,
      reserved: 0,
      reward: 0.1,
      status: 'expired',
      type: 'reviews',
    },
  ],
  userName: 'asd09',
};
