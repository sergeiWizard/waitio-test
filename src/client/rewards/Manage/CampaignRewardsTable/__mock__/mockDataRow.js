// eslint-disable-next-line import/prefer-default-export
export const initialState = {
  currentItem: {
    _id: '5e620da2fb97af17249e3f95',
    activation_permlink: 'activate-monterey-ln7osqo19e',
    budget: 0.1,
    completed: 0,
    name: 'TstCampa',
    remaining: 1,
    reserved: 0,
    reward: 0.1,
    status: 'active',
    type: 'reviews',
  },
  userName: 'asd09',
};
