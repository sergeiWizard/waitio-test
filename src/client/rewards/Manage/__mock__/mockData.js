// eslint-disable-next-line import/prefer-default-export
export const initialState = {
  userName: 'asd09',
  user: {
    active: {},
    balance: '118.710 STEEM',
    can_vote: true,
    name: 'asd09',
  },
};

export const initialStateToDidMount = {
  campaigns: [],
  budgetTotal: {},
};

export const stateToComponentDidMount = {
  budgetTotal: {
    remaining: 6.95,
    sum_budget: 6.96,
    sum_payable: 0.01,
    sum_reserved: 0,
  },
  campaigns: [
    {
      _id: '5e678a1ff9c2cb174722106c',
      activation_permlink: 'activate-monterey-hkvru6omohm',
      budget: 0.01,
      completed: 1,
      name: 'CapaZeka',
      remaining: 0,
      reserved: 0,
      reward: 0.01,
      status: 'active',
      type: 'reviews',
    },
    {
      id: '5e620da2fb97af17249e3f95',
      activation_permlink: 'activate-monterey-ln7osqo19e',
      budget: 0.01,
      completed: 0,
      name: 'TstCampa',
      remaining: 0,
      reserved: 0,
      reward: 0.1,
      status: 'expired',
      type: 'reviews',
    },
  ],
};
