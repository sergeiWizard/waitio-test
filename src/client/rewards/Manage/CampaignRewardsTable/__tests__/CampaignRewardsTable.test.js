import React from 'react';
import { mountWithIntl } from 'enzyme-react-intl';
import { shallow } from 'enzyme';
import { MemoryRouter as Router } from 'react-router-dom';
import CampaignRewardsTable from '../CampaignRewardsTable';
import { initialState } from '../__mock__/mockData';

describe('CampaignRewardsTable', () => {
  let wrapper;

  beforeEach(() => {
    const props = initialState;
    wrapper = mountWithIntl(
      <Router>
        <CampaignRewardsTable {...props} />
      </Router>,
    );
  });
  afterEach(() => jest.clearAllMocks());

  it('should return snapshot', () => {
    const props = initialState;
    wrapper = shallow(
      <Router>
        <CampaignRewardsTable {...props} />
      </Router>,
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should found class of table Campaign-rewards', () => {
    const container = wrapper.find('.Campaign-rewards').first();
    expect(container).toHaveLength(1);
  });
});
