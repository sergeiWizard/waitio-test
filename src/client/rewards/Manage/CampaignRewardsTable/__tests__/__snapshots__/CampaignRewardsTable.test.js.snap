// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampaignRewardsTable should return snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MemoryRouter>
    <InjectIntl(CampaignRewardsTable)
      campaigns={
        Array [
          Object {
            "_id": "5e620da2fb97af17249e3f95",
            "activation_permlink": "activate-monterey-ln7osqo19e",
            "budget": 0.1,
            "completed": 0,
            "name": "TstCampa",
            "remaining": 1,
            "reserved": 0,
            "reward": 0.1,
            "status": "active",
            "type": "reviews",
          },
          Object {
            "_id": "5e38165d31b1bb6e49b265b6",
            "activation_permlink": "activate-monterey-4znrq130xhe",
            "budget": 0.2,
            "completed": 0,
            "name": "capa",
            "remaining": 0,
            "reserved": 0,
            "reward": 0.1,
            "status": "expired",
            "type": "reviews",
          },
          Object {
            "_id": "5e3815b431b1bb6e49b265b4",
            "activation_permlink": "activate-monterey-4p0t3jnbpmr",
            "budget": 0.1,
            "completed": 0,
            "name": "capaz1",
            "remaining": 0,
            "reserved": 0,
            "reward": 0.1,
            "status": "expired",
            "type": "reviews",
          },
        ]
      }
      userName="asd09"
    />
  </MemoryRouter>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <InjectIntl(CampaignRewardsTable)
        campaigns={
          Array [
            Object {
              "_id": "5e620da2fb97af17249e3f95",
              "activation_permlink": "activate-monterey-ln7osqo19e",
              "budget": 0.1,
              "completed": 0,
              "name": "TstCampa",
              "remaining": 1,
              "reserved": 0,
              "reward": 0.1,
              "status": "active",
              "type": "reviews",
            },
            Object {
              "_id": "5e38165d31b1bb6e49b265b6",
              "activation_permlink": "activate-monterey-4znrq130xhe",
              "budget": 0.2,
              "completed": 0,
              "name": "capa",
              "remaining": 0,
              "reserved": 0,
              "reward": 0.1,
              "status": "expired",
              "type": "reviews",
            },
            Object {
              "_id": "5e3815b431b1bb6e49b265b4",
              "activation_permlink": "activate-monterey-4p0t3jnbpmr",
              "budget": 0.1,
              "completed": 0,
              "name": "capaz1",
              "remaining": 0,
              "reserved": 0,
              "reward": 0.1,
              "status": "expired",
              "type": "reviews",
            },
          ]
        }
        userName="asd09"
      />,
      "history": Object {
        "action": "POP",
        "block": [Function],
        "canGo": [Function],
        "createHref": [Function],
        "entries": Array [
          Object {
            "hash": "",
            "key": "u2j4pk",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
        ],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "index": 0,
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "key": "u2j4pk",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "campaigns": Array [
          Object {
            "_id": "5e620da2fb97af17249e3f95",
            "activation_permlink": "activate-monterey-ln7osqo19e",
            "budget": 0.1,
            "completed": 0,
            "name": "TstCampa",
            "remaining": 1,
            "reserved": 0,
            "reward": 0.1,
            "status": "active",
            "type": "reviews",
          },
          Object {
            "_id": "5e38165d31b1bb6e49b265b6",
            "activation_permlink": "activate-monterey-4znrq130xhe",
            "budget": 0.2,
            "completed": 0,
            "name": "capa",
            "remaining": 0,
            "reserved": 0,
            "reward": 0.1,
            "status": "expired",
            "type": "reviews",
          },
          Object {
            "_id": "5e3815b431b1bb6e49b265b4",
            "activation_permlink": "activate-monterey-4p0t3jnbpmr",
            "budget": 0.1,
            "completed": 0,
            "name": "capaz1",
            "remaining": 0,
            "reserved": 0,
            "reward": 0.1,
            "status": "expired",
            "type": "reviews",
          },
        ],
        "userName": "asd09",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <InjectIntl(CampaignRewardsTable)
          campaigns={
            Array [
              Object {
                "_id": "5e620da2fb97af17249e3f95",
                "activation_permlink": "activate-monterey-ln7osqo19e",
                "budget": 0.1,
                "completed": 0,
                "name": "TstCampa",
                "remaining": 1,
                "reserved": 0,
                "reward": 0.1,
                "status": "active",
                "type": "reviews",
              },
              Object {
                "_id": "5e38165d31b1bb6e49b265b6",
                "activation_permlink": "activate-monterey-4znrq130xhe",
                "budget": 0.2,
                "completed": 0,
                "name": "capa",
                "remaining": 0,
                "reserved": 0,
                "reward": 0.1,
                "status": "expired",
                "type": "reviews",
              },
              Object {
                "_id": "5e3815b431b1bb6e49b265b4",
                "activation_permlink": "activate-monterey-4p0t3jnbpmr",
                "budget": 0.1,
                "completed": 0,
                "name": "capaz1",
                "remaining": 0,
                "reserved": 0,
                "reward": 0.1,
                "status": "expired",
                "type": "reviews",
              },
            ]
          }
          userName="asd09"
        />,
        "history": Object {
          "action": "POP",
          "block": [Function],
          "canGo": [Function],
          "createHref": [Function],
          "entries": Array [
            Object {
              "hash": "",
              "key": "u2j4pk",
              "pathname": "/",
              "search": "",
              "state": undefined,
            },
          ],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "index": 0,
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "key": "u2j4pk",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "campaigns": Array [
            Object {
              "_id": "5e620da2fb97af17249e3f95",
              "activation_permlink": "activate-monterey-ln7osqo19e",
              "budget": 0.1,
              "completed": 0,
              "name": "TstCampa",
              "remaining": 1,
              "reserved": 0,
              "reward": 0.1,
              "status": "active",
              "type": "reviews",
            },
            Object {
              "_id": "5e38165d31b1bb6e49b265b6",
              "activation_permlink": "activate-monterey-4znrq130xhe",
              "budget": 0.2,
              "completed": 0,
              "name": "capa",
              "remaining": 0,
              "reserved": 0,
              "reward": 0.1,
              "status": "expired",
              "type": "reviews",
            },
            Object {
              "_id": "5e3815b431b1bb6e49b265b4",
              "activation_permlink": "activate-monterey-4p0t3jnbpmr",
              "budget": 0.1,
              "completed": 0,
              "name": "capaz1",
              "remaining": 0,
              "reserved": 0,
              "reward": 0.1,
              "status": "expired",
              "type": "reviews",
            },
          ],
          "userName": "asd09",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
  Symbol(enzyme.__childContext__): null,
}
`;
